### ISSUE TITLE
Warnings generated when using latest release of Gradle (5.4.1)

### ISSUE THREAD

**Issue by xenomachina** [JLLeitschuh/ktlint-gradle#246]
After upgrading to Gradle 5.4.1, the following warnings are being generated by Gradle:

```
$ ./gradlew --warning-mode all clean check

> Task :ktlintMainSourceSetCheck
The DefaultTask.newOutputFile() method has been deprecated. This is scheduled to be removed in Gradle 6.0. Please use the ObjectFactory.fileProperty() method instead.

> Task :compileKotlin
The ProjectLayout.fileProperty() method has been deprecated. This is scheduled to be removed in Gradle 6.0. Please use the ObjectFactory.fileProperty() method instead.

BUILD SUCCESSFUL in 19s
7 actionable tasks: 7 executed
```

Removing the application of the ktlint-gradle plugin causes both of these warnings to disappear.

**Comment by JLLeitschuh** [comment]
Hi @xenomachina,

Please ensure you're using the latest version of the plugin, I believe that this has been fixed.

**Comment by Tapchicoma** [comment]
Can't reproduce it on my personal project that uses Gradle `5.4.1`. Please, ensure that you are using latest released version of the plugin, which is [8.0.0](https://github.com/JLLeitschuh/ktlint-gradle/blob/master/CHANGELOG.md#800---2019-05-06)

**Comment by xenomachina** [comment]
@JLLeitschuh and @Tapchicoma, you are correct. The repo I was seeing this was still using an old version. I have corrected that. Sorry for the dupe, and thanks!
